Students/Pontinteu Crinela/Assignement1_1/src/assign/pack/Main.java

```diff
@@ -0,0 +1,32 @@
+package assign.pack;
+
+import java.util.*;
+
+public class Main {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		int sum = 0;
+		int givenNr;
+
+		Scanner in = new Scanner(System.in);
+		System.out.println("Input the number ");
+		givenNr = in.nextInt();
+
+		int endLimit;
+		endLimit = (givenNr - 1) / 3;
```

Comment:
Check out [this post](http://math.stackexchange.com/questions/9259/find-the-sum-of-all-the-multiples-of-3-or-5-below-1000) for a more efficient approach.